doc/release-notes.md

```diff
@@ -31,55 +31,103 @@ file to ensure that you do not lose access to existing funds; EXISTING FUNDS
+  Additional arguments to this API may be provided to request the address to be created
+  with a user-specified set of receiver types and diversifier index.
+- `z_getbalanceforaccount` This API makes it possible to obtain balance information
+  on a per-account basis.
+- `z_getbalanceforviewingkey` This newly created API allows a user to obtain
+  balance information for funds visible to a Sapling or Unified full viewing key; if a
+  Sprout viewing key is provided, this method allows retrieval of the balance only in the
+  case that the wallet controls the corresponding spending key.  This API has been added
+  to supplement (and largely supplant) `z_getbalance`. Querying for balance by a single
+  address returns only the amount received by that address, and omits value sent to other
+  diversified addresses derived from the same full viewing key; by using
+  `z_getbalanceforviewingkey` it is possible to obtain a correct balance that includes all
+  amounts controlled by a single spending key, including both those sent to external
+  diversified addresses and to wallet-internal change addresses.
+- `z_listunifiedreceivers` This API allows the caller to extract the individual
+  component receivers from a unified address. This is useful if one needs to
+  provide a bare Sapling or p2pkh address to a service that does not yet
+  support unified addresses.
 
 RPC Changes
 -----------
 
+- The result type for the `listaddresses` endpoint has been modified:
+  - The `keypool` source type has been removed; it was reserved but not used.
+  - In the `sapling` address results, the `zip32AccountId` attribute has been
+    removed in favor of `zip32KeyPath`. This is to allow distinct key paths to
+    be reported for addresses derived from the legacy account under different
+    child spending authorities, as are produced by `z_getnewaddress`.
 - The results of the 'dumpwallet' and 'z_exportwallet' RPC methods have been modified
   to now include the wallet's newly generated emergency recovery phrase as part of the
   exported data.
-
-- The results of the 'getwalletinfo' RPC have been modified to return two new fields:
+- The results of the `getwalletinfo` RPC have been modified to return two new fields:
   `mnemonic_seedfp` and `legacy_seedfp`, the latter of which replaces the field that
-  was previously named `seedfp`. 
-
-Wallet
-------
+  was previously named `seedfp`.
+- A new `pool` attribute has been added to each element returned by `z_listunspent`
+  to indicate which value pool the unspent note controls funds in.
+- `z_listreceivedbyaddress` a `pool` attribute has been added to each result to indicate
+  what pool the received funds are held in.
+- `z_viewtransaction` has been updated to include attributes that provide
+  information about Orchard components of the transaction. Also, the `type`
+  attribute for spend and output values has been deprecated and replaced
+  by the `pool` attribute.
```

Comment:
I think this would have been clearer to say "a `pool` attribute has been added to the output of the following RPC methods:" and then list them.